hadoop-hdds/docs/content/feature/Observability.md

```diff
@@ -97,8 +97,8 @@ Repeat the same for [Object Metrics](https://raw.githubusercontent.com/apache/oz
 ## Distributed tracing
 
 Distributed tracing can help to understand performance bottleneck with visualizing end-to-end performance.
-
-Ozone uses [jaeger](https://jaegertracing.io) tracing library to collect traces which can send tracing data to any compatible backend (Zipkin, ...).
+Ozone makes use of [OpenTelemetry](https://opentelemetry.io/) API for tracing and uses otlp with Grpc format for sending traces.
```

Comment:
Optional, but maybe we can also mention Ozone migrated from OpenTracing to OpenTelemetry in 2.1 (?) for historical context.